examples/misc/multiprocess_sgskip.py

```diff
@@ -43,30 +49,26 @@ def __init__(self):
     def terminate(self):
         plt.close('all')
 
-    def poll_draw(self):
```

Comment:
Huh, what was the point of this? Some kind of closure effect?

Reply:
not sure, but AFAICT it works without it...